server/src/main/scala/com/redi/scalademo/business/DefaultCalculator.scala

```diff
@@ -0,0 +1,23 @@
+package com.redi.scalademo.business
+
+import scala.collection.mutable.ArrayBuffer
+
+class DefaultCalculator extends Calculator {
+
+  override def add(augend: String, addend: String): ValidatedResult[BigDecimal] = {
+    val validator = new NumericStringValidator
```

Comment:
Yep I meant to do that once I created the modules, but I missed this line. Fixed.